plugins/aerial-tick-colors

```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/JadeGerbitz/Aeirial-Tick-Colors.git
```

Comment:
This is just a nit-pick but the repo url probably should be the post name-changed version
```suggestion
repository=https://github.com/JadeGerbitz/Aerial-Tick-Colors.git
```